src/types/time.d.ts

```diff
@@ -0,0 +1,20 @@
+type ParsedDoyString = {
+  doy: number;
+  hour: number;
+  min: number;
+  ms: number;
+  sec: number;
+  time: string;
+  year: number;
+};
+
+type ParsedDateString = {
```

Comment:
I'm wondering if we should call this `ParsedYmdString` for Year Month Day? Just to try and squeeze a little more context from the name.